spec/domain/finders/content_spec.rb

```diff
@@ -235,36 +235,56 @@
   end
 
   describe 'Order' do
-    before do
-      edition1 = create :edition, title: 'last', organisation_id: primary_org_id
-      edition2 = create :edition, title: 'middle', organisation_id: primary_org_id
-      edition3 = create :edition, title: 'first', organisation_id: primary_org_id
+    context 'when values do not repeat' do
+      before do
+        edition1 = create :edition, title: 'last', organisation_id: primary_org_id
+        edition2 = create :edition, title: 'middle', organisation_id: primary_org_id
+        edition3 = create :edition, title: 'first', organisation_id: primary_org_id
 
-      create :metric, edition: edition1, date: 15.days.ago, upviews: 1, feedex: 3
-      create :metric, edition: edition2, date: 15.days.ago, upviews: 2, feedex: 2
-      create :metric, edition: edition3, date: 15.days.ago, upviews: 3, feedex: 1
-      recalculate_aggregations!
-    end
+        create :metric, edition: edition1, date: 15.days.ago, upviews: 1, feedex: 3
+        create :metric, edition: edition2, date: 15.days.ago, upviews: 2, feedex: 2
+        create :metric, edition: edition3, date: 15.days.ago, upviews: 3, feedex: 1
+        recalculate_aggregations!
+      end
 
-    it 'defaults order by descending unique pageviews' do
-      response = described_class.call(filter: filter)
+      it 'defaults order by descending unique pageviews' do
+        response = described_class.call(filter: filter)
 
-      titles = response.fetch(:results).map { |result| result.fetch(:title) }
-      expect(titles).to eq(%w(first middle last))
-    end
+        titles = response.fetch(:results).map { |result| result.fetch(:title) }
+        expect(titles).to eq(%w(first middle last))
+      end
 
-    it 'order by other attribute' do
-      response = described_class.call(filter: filter.merge(sort_attribute: 'feedex'))
+      it 'order by other attribute' do
+        response = described_class.call(filter: filter.merge(sort_attribute: 'feedex'))
 
-      titles = response.fetch(:results).map { |result| result.fetch(:title) }
-      expect(titles).to eq(%w(last middle first))
+        titles = response.fetch(:results).map { |result| result.fetch(:title) }
+        expect(titles).to eq(%w(last middle first))
+      end
+
+      it 'order in ascending' do
+        response = described_class.call(filter: filter.merge(sort_direction: 'asc'))
+
+        titles = response.fetch(:results).map { |result| result.fetch(:title) }
+        expect(titles).to eq(%w(last middle first))
+      end
     end
 
-    it 'order in ascending' do
-      response = described_class.call(filter: filter.merge(sort_direction: 'asc'))
+    context 'when values repeat' do
+      before do
+        edition1 = create :edition, title: 'middle', warehouse_item_id: 'w2', organisation_id: primary_org_id
```

Comment:
So its less confusing can we use `w1` and `w2` as warehouse_item_ids and `first` and `last` as titles?

Reply:
The `warehouse_item_id` is not part of the finder response, so asserting on the `title` felt like a sensible way to go. 

I am not very keen to change what the finder returns in order to make a test more readable.

Reply:
Sorry - Just in the scope of this test - I meant it's not clear why edition1's title is `middle` rather than `first` and why we start counting warehouse_item_ids at `w2`.

```ruby
edition1 = create :edition, title: 'first', warehouse_item_id: 'w1', organisation_id: primary_org_id
edition2 = create :edition, title: 'last', warehouse_item_id: 'w2', organisation_id: primary_org_id
```

Reply:
I have renamed them to `first`, `last` and `w1`, `w2`